interface/themes/tabs_style_full.css

```diff
@@ -360,6 +360,8 @@ z-index:10000;
     color: #333;
     padding: 15px 10px;
     font-weight: 700;
+    transition: background-color 0.5s ease;
```

Comment:
I recommend this be significantly faster, `0.1s` is where I would put it. I think `0.5s` is too slow for an EMR workflow.